### DISCUSSION TITLE
Cannot Use Commands in Voice Channel

### DISCUSSION THREAD

**Discussion by Vaxzel** [jagrosh/MusicBot#1360] [Questions] [answered]
Sorry if I do something wrong.. But I already try all, but the bot still cannot read the message in voice channel..
I already do restart every I make change to the bot in dev portal, re-invite the bot, give it administrator, etc, but not work to read message in voice channel..
Is this a bug?

This is my bot settings in dev portal..

![b1](https://user-images.githubusercontent.com/110272391/233854963-32668228-42e1-467e-9691-75344f0ab5cc.png)

![b2](https://user-images.githubusercontent.com/110272391/233854971-5292f8ef-4ccd-4f8a-a584-96d129c16c91.png)

![b3](https://user-images.githubusercontent.com/110272391/233854973-64d94cd0-4e3f-42c8-a255-f1258d27f8b2.png)

**Comment by MichailiK** [comment]
JMusicBot cannot read Text in Voice Channels yet.

**Reply by Vaxzel** [reply]
So we cannot do that like Taka, Probot, and Vexera?

**Reply by MichailiK** [reply, marked as answer]
No, JMusicBot’s code hasn’t been updated in a while. It has no concept of text in voice channels.

**Reply by Vaxzel** [reply]
Okay.. thanks for your answer...
